tools/cpp/unix_cc_configure.bzl

```diff
@@ -273,7 +273,15 @@ def _is_clang(repository_ctx, cc):
 def _is_gcc(repository_ctx, cc):
     # GCC's version output uses the basename of argv[0] as the program name:
     # https://gcc.gnu.org/git/?p=gcc.git;a=blob;f=gcc/gcc.cc;h=158461167951c1b9540322fb19be6a89d6da07fc;hb=HEAD#l8728
-    return repository_ctx.execute([cc, "--version"]).stdout.startswith("gcc ")
+    cc_argv0 = repository_ctx.execute([cc, "--version"]).stdout.partition(" ")[0]
+    if cc_argv0 == "gcc":
```

Comment:
Please change to `x.startswith("gcc ") or x.startswith("gcc-")` without using call to `partition` first.

Check without using `partition` should be more robust. One thing less that can go wrong. (But I'm also nitpicking a bit).

Reply:
Thanks for having a look! Rebased and updated. :)